src/website/app/demos/Button/examples/rtl.js

```diff
@@ -21,8 +21,9 @@ import IconBackspace from '../../../../../Icon/IconBackspace';
 
 export default {
   id: 'rtl',
-  title: 'RTL Support',
-  description: `Buttons with Icons are reversed when the \`rtl\` prop is set.
+  title: 'Bidirectionality',
+  description: `Buttons support right-to-left (RTL) languages.
+Buttons with Icons are reversed when the \`direction\` prop is set to \`rtl\`.
```

Comment:
Button doesn't have a `direction` prop. Did you mean the theme variable?

Reply:
whoa, changed